src/SwooleRequestHandlerRunner.php

```diff
@@ -329,6 +329,5 @@ private function emitMarshalServerRequestException(
     ) : void {
         $response = ($this->serverRequestErrorResponseGenerator)($exception);
         $emitter->emit($response);
-        $this->logger->logAccessForPsr7Resource($request, $psr7Response);
```

Comment:
It would be better to do the following:

- `s/psr7Response/response/`, as that value IS a PSR-7 response.
- Generate an empty `ServerRequest` instance for `$request`, _or_ allow that value to be nullable in `logAccessForPsr7Resource()`.

Essentially, we want to log the errors when possible.

Reply:
@weierophinney `logAccessForPsr7Resource()` wants a `SwooleHttpRequest`, not `ServerRequest`。